src/magic_folder/test/test_magic_folder_service.py

```diff
@@ -35,15 +44,34 @@
     MagicFolder,
     LocalSnapshotService,
 )
+from ..magicpath import (
+    path2magic,
+)
+from ..config import (
+    create_global_configuration,
+)
+from ..cli import (
+    TahoeClient,
+)
+
+from ..testing.web import (
+    create_fake_tahoe_root,
+    create_tahoe_treq_client,
+)
 
 from .common import (
     SyncTestCase,
 )
 from .strategies import (
     relative_paths,
+    path_segments,
+    local_authors,
+    tahoe_lafs_dir_capabilities,
+    tahoe_lafs_readonly_dir_capabilities,
+    folder_names,
 )
 from .test_local_snapshot import (
-    MemorySnapshotCreator,
+    MemorySnapshotCreator as LocalMemorySnapshotCreator,
```

Comment:
Is there a need for this aliasing? If `MemorySnapshotCreator` doesn't conflict, it might be easier to search etc if the name stays the same..

Reply:
Yes, there is no need. I also had the `MemorySnapshotCreator` from `test_upload` module imported but it was removed in a later refactoring. But the above code remained.